quinn-proto/src/connection/streams/mod.rs

```diff
@@ -443,6 +446,24 @@ pub struct ClosedStream {
     _private: (),
 }
 
+impl ClosedStream {
+    #[doc(hidden)] // For use in quinn only
+    pub fn new() -> Self {
+        Self { _private: () }
+    }
+
+    /// Returns whether the stream is closed
+    pub fn is_closed(&self) -> bool {
```

Comment:
What's this for?

Reply:
Not sure! Appears to be unused, looks like a bit of debris from an earlier refactoring. Will deprecate.

Reply:
It looks to me like it's newly added in this commit.

Reply:
Oh, huh.

Reply:
Killed it.